### ISSUE TITLE
VsCode Can't enter Chinese when use cmd+L Add Highlighted Code

### ISSUE THREAD

**Issue by xielingdm** [continuedev/continue#3199]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: macos and windows
- Continue version: v0.9.239 
- IDE version: 19.2
- Model: gemini
- config.json:
```

### Description

cmd+L highlighted code Can't input Chinese, but English is ok

https://github.com/user-attachments/assets/6476fbe5-42dc-43db-b54b-a624366d3dbe

**Comment by wangshixiang** [comment]
0.8.59 works well, uncheck the auto update checkbox,then install this version

**Comment by Patrick-Erichsen** [comment]
Hi @xielingdm , is this only happening when you've added code using cmd+l?

**Comment by wangshixiang** [comment]
> Hi @xielingdm , is this only happening when you've added code using cmd+l?

yes, only after added code. since version 8.0.60

**Comment by SuiYunsy** [comment]
I got this, too. Ctrl + L and then cannot input Chinese. 😢

**Comment by Patrick-Erichsen** [comment]
This is only after using `Cmd + L`? We've had issues with Chinese character input on JetBrains but I don't believe we've had issues on VS Code.

cc @sestinj here as well

**Comment by sestinj** [comment]
You should be able to type after adding just a single space or a new line. Can anyone confirm whether or not this workaround works?

**Comment by qh0814** [comment]
> You should be able to type after adding just a single space or a new line. Can anyone confirm whether or not this workaround works?

Adding a space or a new line is effective, but this must be done from the beginning. Otherwise, I have to redo the Cmd + L process.

**Comment by wangshixiang** [comment]
yes, in version 0.9.245, a space or new line is help, but not convenient

**Comment by Patrick-Erichsen** [comment]
We will be taking a closer look into this in the next few days 👍

**Comment by Patrick-Erichsen** [comment]
Bisected this down to the following commit (by me): https://github.com/continuedev/continue/commit/b4d364f7616a7bed85469d8655ff048b108aa786#r150215329

@goWrongWay - apologies for not seeing your comment earlier! Changing the underlying HTML tag that is used to represent our codeblocks was the root cause of the issue. 

This will be fixed in our next pre-release 👍  thank you all for bringing this to our attenting!

**Comment by x1ntt** [comment]
Thanks for the timely fix, I no longer have this problem in v0.9.250 🫡

**Comment by raonsol** [comment]
Is it fixed now? I have same problem in version 0.8.66 with Korean characters.
